docs/new_tests.rst

```diff
@@ -0,0 +1,53 @@
+.. _topics-new_tests:
+
+=========
+New Tests
+=========
+
+Writing ducktape Tests
+======================
+
+Subclass ``ducktape.tests.test.Test and implement`` as many ``test`` methods as you
+want. The name of each test method must start or end with ``test``,
+e.g. ``test_functionality`` or ``example_test``. Typically, a test will
+start a few services, collect and/or validate some data, and then finish.
+
+If the test method finishes with no exceptions, the test is recorded as successful, otherwise it is recorded as a failure.
+
+
+Here is an example of a test that just starts a Zookeeper cluster with 2 nodes, and a
+Kafka cluster with 3 nodes::
+
+    class StartServicesTest(Test):
+        """Make sure we can start Kafka and Zookeeper services."""
+        def __init__(self, test_context):
+            super(StartServicesTest, self).__init__(test_context=test_context)
+            self.zk = ZookeeperService(test_context, num_nodes=2)
+            self.kafka = KafkaService(test_context, num_nodes=3, self.zk)
+
+        def test_services_start(self):
+            self.zk.start()
+            self.kafka.start()
+
+Test Parameters
+===============
+
+Use test decorators to parametrize tests, examples are provided below
+
+.. autofunction:: ducktape.mark._mark.parametrize
```

Comment:
In `__init__.py` we import the stuff from `_mark` such that it will be available without the underscore, e.g. via `ducktape.mark.parametrize` instead of `ducktape.mark._mark.parametrize`. Since we don't really want to use the latter import, will the `autofunction` handle the shorter, cleaner version here?